awscrt/common.py

```diff
@@ -18,3 +18,10 @@ def get_cpu_count_for_group(group_idx: int) -> int:
     Returns number of processors in a given group.
     """
     return _awscrt.get_cpu_count_for_group(group_idx)
+
+
+def install_crash_handler():
```

Comment:
Seems easier to just query an environment variable from C
Vs this thing that customers need to call, which (means it can't catch initialization crashes, because initialization has already happend). And we need to worry about it being called multiple times. And how would a CLI user make this happen?

Reply:
Thanks, fixed.